### DISCUSSION TITLE
Change installation directory

### DISCUSSION THREAD

**Discussion by poggingfish** [idena-network/idena-desktop#724] [Q&A]
I am running Windows 11 64bit on my computer and i am wondering how i can change the installation directory of Idena Desktop because my main C: drive does not have enough space for a node. Thanks :)

**Comment by rioda-org** [comment]
https://idena.site/faq/change-default-location-for-idena-node-in-windows-app

1. Quit the App
2. Move the node folder (%appdata%/idena/node) to D:\ or wherever you want
3. Run the node manually, node/idena-go.exe
4. Run App and connect it to the running node
Settings > Node > Connect to remote node, find api key in node/datadir/api.key file
